libreflect/src/stack_setup.c

```diff
@@ -96,16 +96,16 @@ void stack_setup(size_t *stack_base, int argc, char **argv, char **env, size_t *
 	dprint("  0x%08zx\n", stack_base[1 + ii]);
 
 	for (ii = 0; env[ii]; ii++) {
-		stack_base[1 + argc + ii] = (size_t)env[ii];
-		dprint("  0x%08zx\n", stack_base[1 + argc + ii]);
+		stack_base[2 + argc + ii] = (size_t)env[ii];
```

Comment:
For clarity of what's going on (and to hopefully discourage other off-by-one-errors, I would recommend:
```suggestion
		stack_base[1+ argc + ii + 1] = (size_t)env[ii];
```